sched/scheduler/task_runner.go

```diff
@@ -10,23 +11,23 @@ import (
 // Run the task on the specified worker, and update the SagaLog appropriately.  Returns an error if one
 // occurs while running a task or writing to the SagaLog.  This method blocks until all saga messages
 // are logged and the task completes
-func runTaskAndLog(saga *saga.Saga, worker worker.Worker, taskId string, task sched.TaskDefinition) error {
+func runTaskAndLog(saga *saga.Saga, worker worker.Worker, taskId string, task sched.TaskDefinition) (runner.ProcessStatus, error) {
```

Comment:
Ok. I did want an overall failure rate but I suppose we can just add them up in our dashboard.